sanskrit_parser/util/normalization.py

```diff
@@ -0,0 +1,51 @@
+'''
+Handles user-input and converts it into the format that internal modules expect
+and vice-versa
+
+@author:
+'''
+
+from __future__ import print_function, unicode_literals
```

Comment:
Should we add a few tests for this module?

Reply:
Definitely! I did not get around to it. Please go ahead if you are interested.

Reply:
Sure, can pick this up after tests related tasks (if it's open).

Reply:
Opened #75 to keep track